Contributing.md

```diff
@@ -0,0 +1,92 @@
+# Contribution Guide
+
+If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🥳
+
```

Comment:
Specify here that we only recommend doing local setup in a Linux environment. We will be soon updating the readme for Windows setup.